core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -152,7 +152,7 @@ private[deploy] class Master(
       logInfo(s"Spark Master is acting as a reverse proxy. Master, Workers and " +
        s"Applications UIs are available at $masterWebUiUrl")
     }
-    checkForWorkerTimeOutTask = forwardMessageThread.scheduleAtFixedRate(
+    checkForWorkerTimeOutTask = forwardMessageThread.scheduleWithFixedDelay(
       () => Utils.tryLogNonFatalError { self.send(CheckForWorkerTimeOut) },
```

Comment:
+CC @Ngone51 Any thoughts on how this will impact standalone ? For changes to both `Master` and `Worker` classes.
Typically I would not prefer a behavior change for a cleanup PR.